platform-configs/rpi-3b.robot

```diff
@@ -0,0 +1,272 @@
+${RELEASE_DATE_FROM_SOL}=                           ${FALSE}
+${MANUFACTURER_VERIFICATION}=                       ${FALSE}
+${VENDOR_VERIFICATION}=                             ${FALSE}
+${FAMILY_VERIFICATION}=                             ${FALSE}
+${TYPE_VERIFICATION}=                               ${FALSE}
+${HARDWARE_WP_SUPPORT}=                             ${FALSE}
+${DOCKING_STATION_USB_SUPPORT}=                     ${FALSE}
+${DOCKING_STATION_KEYBOARD_SUPPORT}=                ${FALSE}
+${DOCKING_STATION_USB_C_CHARGING_SUPPORT}=          ${FALSE}
+${DOCKING_STATION_DETECT_SUPPORT}=                  ${FALSE}
+${DOCKING_STATION_AUDIO_SUPPORT}=                   ${FALSE}
+${EMMC_SUPPORT}=                                    ${FALSE}
+${DTS_SUPPORT}=                                     ${FALSE}
+${FIRMWARE_BUILDING_SUPPORT}=                       ${FALSE}
+${CPU_TESTS_SUPPORT}=                               ${FALSE}
+${DOCKING_STATION_NET_INTERFACE}=                   ${FALSE}
+${DOCKING_STATION_HDMI}=                            ${FALSE}
+${DOCKING_STATION_DISPLAY_PORT}=                    ${FALSE}
+${UPLOAD_ON_USB_SUPPORT}=                           ${FALSE}
+${DOCKING_STATION_UPLOAD_SUPPORT}=                  ${FALSE}
+${THUNDERBOLT_DOCKING_STATION_SUPPORT}=             ${FALSE}
+${THUNDERBOLT_DOCKING_STATION_USB_SUPPORT}=         ${FALSE}
+${THUNDERBOLT_DOCKING_STATION_KEYBOARD_SUPPORT}=    ${FALSE}
+${THUNDERBOLT_DOCKING_STATION_UPLOAD_SUPPORT}=      ${FALSE}
+${THUNDERBOLT_DOCKING_STATION_NET_INTERFACE}=       ${FALSE}
+${THUNDERBOLT_DOCKING_STATION_HDMI}=                ${FALSE}
+${THUNDERBOLT_DOCKING_STATION_DISPLAY_PORT}=        ${FALSE}
+${THUNDERBOLT_DOCKING_STATION_AUDIO_SUPPORT}=       ${FALSE}
+${DOCKING_STATION_SD_CARD_READER_SUPPORT}=          ${FALSE}
+${RESET_TO_DEFAULTS_SUPPORT}=                       ${FALSE}
+
+# Test module: dasharo-security
+${TPM_SUPPORT}=                                     ${FALSE}
+${VBOOT_KEYS_GENERATING_SUPPORT}=                   ${FALSE}
+${VERIFIED_BOOT_SUPPORT}=                           ${FALSE}
+${VERIFIED_BOOT_POPUP_SUPPORT}=                     ${FALSE}
+${MEASURED_BOOT_SUPPORT}=                           ${FALSE}
+${SECURE_BOOT_SUPPORT}=                             ${FALSE}
+${ME_NEUTER_SUPPORT}=                               ${FALSE}
+${USB_STACK_SUPPORT}=                               ${FALSE}
+${USB_MASS_STORAGE_SUPPORT}=                        ${FALSE}
+${TCG_OPAL_DISK_PASSWORD_SUPPORT}=                  ${FALSE}
+${BIOS_LOCK_SUPPORT}=                               ${FALSE}
+${SMM_WRITE_PROTECTION_SUPPORT}=                    ${FALSE}
+${WIFI_BLUETOOTH_CARD_SWITCH_SUPPORT}=              ${FALSE}
+${CAMERA_SWITCH_SUPPORT}=                           ${FALSE}
+${EARLY_BOOT_DMA_SUPPORT}=                          ${FALSE}
+${UEFI_PASSWORD_SUPPORT}=                           ${FALSE}
+
+# Test module: dasharo-performance
+${SERIAL_BOOT_MEASURE}=                             ${FALSE}
```

Comment:
Many of these flags does not make sense for Yocto/RPi testing. We need to come up with a way to manage this, not necessarily in this MR.

Reply:
@WiktorG351 Please at least create issue and link here, so we can close this thread.

Reply:
@macpijan yes, [here](https://github.com/Dasharo/open-source-firmware-validation/issues/33) is the new issue

Reply:
added 1 comment as well - marking as resolved